packages/webview_flutter/webview_flutter/example/lib/main.dart

```diff
@@ -463,6 +532,11 @@ class SampleMenu extends StatelessWidget {
 
     return indexFile.path;
   }
+
+  Future<void> _basicAuthExample() {
+    return webViewController.loadRequest(Uri.parse(
+        'https://www.httpwatch.com/httpgallery/authentication/#showExample10'));
```

Comment:
This example looks great! The only problem is that is uses a third-party website. I think we could actually create our own HTML for this example.

Reply:
HTTP authentication requires a server, not just HTML, which makes it problematic for an example app. For an integration test we can use a local test server, but I don't see a good solution here for controlling an endpoint.

Maybe we should allow pasting in a URL to load for this example, instead of hard-coding something, since I can't find an example page I'd want to put in our app.

Reply:
@stuartmorgan could you elaborate on your proposed solution? Are you suggesting to change the example app so it starts with a textfield element, allowing the user to navigate to any page they desire?